plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/utils/GeneralUtils.java

```diff
@@ -162,6 +166,46 @@ public static byte[] convertToBytes(String strValue) {
         return bytes;
     }
 
+    /**
+     * The function check warning Unicode value in string
+     *
+     */
+    public static boolean containsInValidUnicodeSymbol(@Nullable String strValue) {
+        if (strValue == null || strValue.isEmpty()) {
+            return false;
+        }
+        for (int i = 0; i < strValue.length(); i++) {
+            int code = strValue.codePointAt(i);
+            for (int j = 0; j < INVALID_STRING_CODE.length; j++) {
+                if (code == INVALID_STRING_CODE[j]) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
+
+    /**
+     * The function remove warning Unicode value from input
+     *
+     */
+    public static String removeInValidUnicodeSymbol(@Nullable String inputStr) {
```

Comment:
We need to trim it, not to remove all spaces.
Trim = delete leading and trailing characters.